src/Labels/Labels.php

```diff
@@ -122,6 +124,11 @@ public function getLabels(): array
 			$output = \array_merge($output, $this->getPipedriveLabels());
 		}
 
+		// Calculator.
+		if (UtilsSettingsHelper::isOptionCheckboxChecked(SettingsCalculator::SETTINGS_CALCULATOR_USE_KEY, SettingsCalculator::SETTINGS_CALCULATOR_USE_KEY)) {
```

Comment:
```suggestion
		if ($this->settingsCalculator->isSettingsGlobalValid()) {
```

Just need to add the `SettingsCalculator` as a dependency and reuse the method.

Reply:
we are checking this with filter values to avoid to much dependencies, so this will stay as is